processor/src/main/java/org/mapstruct/ap/internal/model/source/FormattingParameters.java

```diff
@@ -40,4 +40,28 @@ public String getNumber() {
         return number;
     }
 
+    @Override
+    public boolean equals(Object o) {
```

Comment:
No problem. I didn't realize I have committed that as well, I was in a hurry to commit and push then and might have pushed something not needed :)